### PR TITLE
Use matrix for building.

### PR DESCRIPTION
**Pull request by gabrieldemarmiesse**
Some refactoring to help with future changes.

### CODE REVIEW

File: .github/workflows/make_wheel_Windows.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+./bazel-${BAZEL_VERSION}-windows-x86_64.exe build \
+  -c opt \
+  --enable_runfiles \
+  --noshow_progress \
+  --noshow_loading_progress \
+  --verbose_failures \
+  --test_output=errors \
+  build_pip_pkg
+bazel-bin/build_pip_pkg artifacts $NIGHTLY_FLAG
+mv artifacts wheelhouse
```
Comment (seanpmorgan) [inline .github/workflows/make_wheel_Windows.sh]:
```suggestion
bazel-bin/build_pip_pkg wheelhouse $NIGHTLY_FLAG
```
nit: First argument is the output directory

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -29,127 +29,37 @@ jobs:
...
-            --enable_runfiles \
-            --noshow_progress \
-            --noshow_loading_progress \
-            --verbose_failures \
-            --test_output=errors \
-            build_pip_pkg
-          bazel-bin/build_pip_pkg artifacts $NIGHTLY_FLAG
+        run: bash .github/workflows/make_wheel_${OS}.sh
       - uses: actions/upload-artifact@v1
         if: github.event_name == 'schedule' || github.event_name == 'release' || matrix.python-version == '3.5'
```
Comment (seanpmorgan) [inline .github/workflows/release.yml]: Can we remove this check? I believe the upload will have no affect if there are no whl's created in the step prior?

File: .github/workflows/make_wheel_Windows.sh (hunk 1 above)
Comment (gabrieldemarmiesse) [reply .github/workflows/make_wheel_Windows.sh]: Thanks!

File: .github/workflows/release.yml (hunk 2 above)
Comment (gabrieldemarmiesse) [reply .github/workflows/release.yml]: We can always try.

Comment (gabrieldemarmiesse) [reply .github/workflows/release.yml]: I doesn't seem to work. I'll revert the commit.

**Pull request merged**